modules/images/webp-uploads/image-edit.php

```diff
@@ -320,10 +320,39 @@ function webp_uploads_get_next_full_size_key_from_backup( $attachment_id ) {
  * @return array The updated metadata of the attachment.
  */
 function webp_uploads_restore_image( $attachment_id, $data ) {
-	$backup_sources = get_post_meta( $attachment_id, '_wp_attachment_backup_sources', true );
+	if ( empty( $data['sources'] ) ) {
+		return $data;
+	}
```

Comment:
Good question, the main point of this function is not to set up `sources` if `sources` is not present we can assume the image does not have any available sources. Since this function is being called when the `restore` image is executed. At that point, we are not creating sources but restoring them so is safe to assume here if a `sources` is not present there's nothing to be restored (at least nothing custom created by this plugin) since this plugin is the one creating `sources` property.